server/tables/tables.go

```diff
@@ -347,6 +347,7 @@ type ExecutionNode struct {
 	OS                    string
 	Arch                  string
 	Pool                  string
+	Scheduler             string
```

Comment:
consider renaming to `SchedulerAddress` (or `SchedulerHostPort` since address might be ambiguous as to whether the URL scheme is included)

Reply:
+1 to SchedulerAddress

Reply:
Changed to SchedulerHostPort